lib/pact/command/cont_command.ex

```diff
@@ -215,7 +218,8 @@ defmodule Kadena.Pact.ContCommand do
           keypairs: keypairs
         } = cmd_request
       ) do
-    with {:ok, payload} <- create_payload(cmd_request),
+    with {:ok, cmd_request} <- set_signers_from_keypair(cmd_request),
+         {:ok, payload} <- create_payload(cmd_request),
```

Comment:
@Odraxs Let's think about this use case: a Command that has added signers and key pairs ..... how would the code handle this?

Reply:
In that use case, we should add signatures to the existing list, making use of the Key Pairs??

Reply:
@EdwinGuayacan Here is my suggestion.

1. Keep the functions for adding keypairs and signers as they are.
2. Every time `add_keypair/2` is invoked, build the signer. (prevent duplicates).
3. That's it.

is it clear?

Reply:
@juanhurtado10 is it possible to add the same keypair several times, is that ok or is it also necessary to avoid the keypair being duplicated when using `add_keypair/2` or do we just need to prevent duplicate signers?

Reply:
@Odraxs 🤔 that's a good question. Let's simplify, for this PR let's allow duplicates for both. Then we create a specific issue to handle duplicates.